rocAL/rocAL_pybind/amd/rocAL/fn.py

```diff
@@ -0,0 +1,534 @@
+
+from amd.rocAL import readers
+from amd.rocAL import decoders
+import inspect
+import amd.rocAL.types as types
+import rali_pybind as b
+from amd.rocAL.pipeline import Pipeline
+
+def blend(*inputs,**kwargs):
+    kwargs_pybind = {"input_image0":inputs[0], "input_image1":inputs[1], "is_output":False ,"ratio":None}
+    blend_image = b.Blend(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    return (blend_image)
+
+def snow(*inputs, snow=0.5, device=None):
+    # pybind call arguments
+    kwargs_pybind = {"input_image0": inputs[0],"is_output": False, "shift": None}
+    snow_image = b.Snow(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    return (snow_image)
+
+def exposure(*inputs, exposure=0.5, device=None):
+    # pybind call arguments
+    kwargs_pybind = {"input_image0": inputs[0],"is_output": False, "shift": None}
+    exposure_image = b.Exposure(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    return (exposure_image)
+
+def fish_eye(*inputs, device=None):
+    # pybind call arguments
+    kwargs_pybind = {"input_image0": inputs[0],"is_output": False}
+    fisheye_image = b.FishEye(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    return (fisheye_image)
+
+def fog(*inputs, fog=0.5, device=None):
+    # pybind call arguments
+    kwargs_pybind = {"input_image0": inputs[0],
+                     "is_output": False, "fog_value": None}
+    fog_image = b.Fog(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+    return (fog_image)
+
+def brightness(*inputs, brightness=1.0, bytes_per_sample_hint=0, image_type=0,
```

Comment:
Can you check on the fixed parameters - alpha and beta?

Reply:
I have added brightness_fixed augmentation, it uses alpha & beta params that can be exposed to the user